app/src/marketplace/components/ProductPageEditor/ProductDetailsEditor/index.jsx

```diff
@@ -210,6 +224,23 @@ class ProductDetailsEditor extends React.Component<Props, State> {
                     defaultValue={product.description}
                     onChange={(e: SyntheticInputEvent<EventTarget>) => onEdit('description', e.target.value)}
                 />
+                {setPriceDialogOpen && (
+                    <Modal>
```

Comment:
Should the `<Modal>` tags be moved under the respective compoenents? For example here under `SetPriceDialog`... that way there is no extra import for `Modal` when using them.

Reply:
I remember myself trying to make dialogs usable directly (w/o the extra `Modal`) but ended up fighting props. I think I had to refresh my perspective though (by going through you guys). I may have an idea for it now! 🤠

Reply:
Yep, I was wondering :) Anyway, everything looks good, though 👍

Reply:
I followed your advice and embodied `<Modal>` inside the actual Dialog-components. Looks nicer!